components/analytics/grafana/dashboards/management.json

```diff
@@ -130,7 +143,7 @@
                         }
                     },
                     "queryType": "sql",
-                    "rawSql": "SELECT DATE_TRUNC('day', timestamp) as time,\r\n       COUNT(CASE WHEN obj_val = 'completed' THEN 1 END) as completed,\r\n       COUNT(CASE WHEN obj_val = 'rejected' THEN 1 END) as rejected\r\nFROM events \r\nWHERE $__timeFilter(timestamp)\r\n    AND (scope = 'update:job')\r\n    AND (obj_name = 'status')\r\n    AND (obj_val IN ('completed', 'rejected'))\r\nGROUP BY time\r\nORDER BY time\r\n",
+                    "rawSql": "SELECT DATE_TRUNC('day', timestamp) as time,\r\n       COUNT(CASE WHEN obj_val = 'completed' THEN 1 END) as completed,\r\n       COUNT(CASE WHEN obj_val = 'rejected' THEN 1 END) as rejected\r\nFROM events \r\nWHERE $__timeFilter(timestamp)\r\n    AND (scope = 'update:job')\r\n    AND (obj_name = 'status')\r\n    AND (obj_val IN ('completed', 'rejected'))\r\n    AND (user_name IN ['kyle', 'rpalugod@sewerai.com', 'rswords@sewerai.com'])\r\nGROUP BY time\r\nORDER BY time\r\n",
```

Comment:
Are we suppose to have name or email addresses?

Reply:
Those are the names that are registered in CVAT as usernames, unfortunately.